src/app/component/WishlistItem/WishlistItem.component.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * ScandiPWA - Progressive Web App for Magento
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/base-theme
+ * @link https://github.com/scandipwa/base-theme
+ */
+
+import { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Field from 'Component/Field';
+import ProductCard from 'Component/ProductCard';
+import { ProductType } from 'Type/ProductList';
+import { debounce } from 'Util/Request';
+
+import './WishlistItem.style';
+
+export const UPDATE_WISHLIST_FREQUENCY = 1000; // (ms)
+
+export default class WishlistItem extends PureComponent {
+    static propTypes = {
+        addToCart: PropTypes.func,
+        changeQuantity: PropTypes.func,
+        product: ProductType.isRequired,
+        changeDescription: PropTypes.func,
+        removeFromWishlist: PropTypes.func,
+        getParameters: PropTypes.func.isRequired
+    };
+
+    static defaultProps = {
+        addToCart: () => {},
+        changeQuantity: () => {},
+        changeDescription: () => {},
+        removeFromWishlist: () => {}
+    };
+
+    changeQuantity = debounce((quantity) => {
+        const { product: { wishlist: { id } }, changeQuantity } = this.props;
+        changeQuantity(id, quantity);
+    }, UPDATE_WISHLIST_FREQUENCY);
+
+    changeDescription = debounce((description) => {
+        const { product: { wishlist: { id } }, changeDescription } = this.props;
+        changeDescription(id, description);
+    }, UPDATE_WISHLIST_FREQUENCY);
+
+    addToCart = () => {
+        const { product, addToCart } = this.props;
+        addToCart(product);
+    };
```

Comment:
Can not you simply implement this in props? Take `const { product, addToCart } = this.props;` in container.